bundles/statistics/statsgrid2016/instance.js

```diff
@@ -47,15 +50,24 @@ Oskari.clazz.define(
             // disable tile if we don't have anything to show or enable if we do
             // setup initial state
             this.setState();
+            
+            me.getTile().getFlyoutManager().init();
 
             if(this.isEmbedded()) {
                 // Start in an embedded map mode
-                // Embedded map might not have the grid. If enabled show toggle buttons so user can access it
-                me.showToggleButtons(conf.grid !== false, this.state.view);
+
                 // Always show legend on map when embedded
                 me.showLegendOnMap(true);
                 // Classification can be disabled for embedded map
                 me.enableClassification(conf.allowClassification !== false);
+
+                // TODO? better way to call than strings
+                if( me.conf.grid ) {
+                    me.showEmbeddedTools('table', true);
```

Comment:
Are these "shown" as in toggled on or just added to the ui so user can open them?